src/widgets/search-box/__tests__/search-box-test.js

```diff
@@ -243,7 +245,7 @@ describe('searchBox()', () => {
       simulateInputEvent('test', 'tes', widget, helper, state, container);
 
       // When
-      $('button[type="reset"]')[0].click();
+      $('.ais-search-box--reset-wrapper')[0].click();
```

Comment:
the button is still type reset right?

Reply:
yes. The problem is that it is in the template, so we cannot rely on what's inside it for the tests.

Reply:
This was the source of the bug. If you were to use an emoji, there would not be a button (with type=reset) so instead the event listener is on the outer span.